templates/taxi/index.html

```diff
@@ -10,4 +10,6 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+
+  <p>You have visited this page {{ num_visits }} {% if num_visits == 1 %} time {% else %} times {% endif %}.</p>
```

Comment:
It's correct but some suggestion:
`<p>You have visited this page {{ num_visits }} time{{ num_visits|pluralize }}</p>` (best practice)

Reply:
Thank for this, I've fixed it.